.github/workflows/reusable_dockerfile_pipeline.yml

```diff
@@ -26,6 +31,11 @@ jobs:
           echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
           # yamllint disable
           echo "IMAGE_NAME=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
+          # here we validate if we have specified a different package name in
+          # the inputs, if so, we change the package to it.
+          if [[ ${{ inputs.packageName }} != ${{ github.repository}} ]];then
+            echo "IMAGE_NAME=$(echo ${{ github.repository_owner }}/${{ inputs.packageName }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
```

Comment:
Users could add other characters than the ones supported by a container registry. Maybe check with a short regex like `([A-Za-z0-9\-]+)` with a helpful output.

Reply:
good point!
already done and tested!

https://github.com/jrmanes/docker-cicd/actions/runs/4232871935/jobs/7353129308


cc: @Bidon15